### PR TITLE
[GSoC] Added doit method in symbolic Probability

### PR DESCRIPTION
**Pull request by Smit-create**
Added doit method in symbolic Probability
#### Brief description of what is fixed or changed
`P=probability` now calls `Probability(...).doit()`

#### Release Notes

* stats
    * Added `doit` in class `Probability`

ping @Upabjojr @czgdp1807 @jmig5776

### CODE REVIEW

**Comment by Smit-create** [comment]
@czgdp1807 @Upabjojr  Please have a look if this looks to good to merge.

File: sympy/stats/symbolic_probability.py [hunk 1]
```diff
@@ -173,6 +229,8 @@ def expand(self, **hints):
                 else:
                     nonrv.append(a)
             return Mul(*nonrv)*Expectation(Mul(*rv), condition=condition)
+        elif isinstance(_expand(expr), Add):
+            return Expectation(_expand(expr)).expand()
```
Comment (Upabjojr) [inline sympy/stats/symbolic_probability.py:233]: Calling `_expand(expr)` twice? Furthermore, `.expand()` is expensive... you know you just need to distribute an addition over `Expectation`, so...
```python
r = _expand(expr)
return Add.fromiter(Expectation(i) for i in r)
```

**Comment by czgdp1807** [comment]
Tests passing, coverage is good. LGTM.

**Comment by Smit-create** [comment]
@Upabjojr  Does this look good to merge?

**Comment by Upabjojr** [comment]
> @Upabjojr Does this look good to merge?

Yes, it does!

**Pull request merged**